pom.xml

```diff
@@ -108,7 +108,7 @@
         <maven-source-plugin-version>3.0.1</maven-source-plugin-version>
         <maven-surefire-plugin.version>2.17</maven-surefire-plugin.version>
         <maven-site-plugin-version>3.5</maven-site-plugin-version>
-        <maven-checkstyle-plugin-version>2.17</maven-checkstyle-plugin-version>
+        <maven-checkstyle-plugin-version>2.16</maven-checkstyle-plugin-version>
```

Comment:
I would be in favor of bumping to current if we can, but if it causes other issues to get picked up, don't worry about it right now, we can bump it later.